controller/placeholder.js

```diff
@@ -302,8 +302,7 @@ function setup(placeholderService, do_geometric_filters_apply, should_execute) {
         ];
 
         logger.debug(messageParts.join(' '));
-        debugLog.push(req, messageParts[1].slice(1,-1));
```

Comment:
this `.slice()` command is odd, the purpose appears to be to trim the square brackets off the start and end.
it was somewhat convenient to print the `count` (ie. `results.length`) but that's probably not helpful in practise.